Description: Remove flattr link, which uses non-free icon.
Author: Jelmer Vernooij <jelmer@debian.org>
Status: not forwarded upstream

diff --git a/docs/index.html b/docs/index.html
index 172fbb5..eeb253d 100644
--- a/docs/index.html
+++ b/docs/index.html
@@ -58,8 +58,6 @@
     <p>
         <a href="{{ pathto('contribute') }}">Write Code</a>,
         <a href="https://www.transifex.com/projects/p/isso/">Translate</a> or
-        <a href="https://flattr.com/thing/2059355/posativisso-on-GitHub">Flattr
-        <img src="{{ pathto('_static/flattr.png', 1) }}" alt="Flattr icon"/></a>.
     </p>
 
     {% include "searchbox.html" %}
